lib/server.js

```diff
@@ -8,9 +8,15 @@ module.exports = (Hapi, config, options, requireCwd) => {
       delete serverConfig.cache;
     } else {
       serverConfig.cache.provider.constructor = requireCwd(serverConfig.cache.provider.constructor);
+      if (options.verbose) {
+        console.log('Cache is enabled on this server using the following config:');
+        console.log(serverConfig.cache);
+      }
     }
   }
-  if (process.env.PORT) {
+  if (options.portOverride) {
```

Comment:
not quite right.  maybe lets call it `options.usePortEnv` and default it to true and if its false, don't set port from env var